test/fixtures/transpile-css-prop/code.js

```diff
@@ -92,3 +92,19 @@ const CustomCompWithDot = p => <Button.Ghost css="flex: 1">H</Button.Ghost>
 const NestedCompWithDot = p => (
   <Button.Ghost.New css="flex: 1">H</Button.Ghost.New>
 )
+
+const DynamicComp = p => {
+  const { Box } = p.components;
+
+  <Box css="color: blue" title="Hello">X</Box>
+}
+
+const DynamicCompWithDot = p => {
+  <p.components.Box css="color: blue" title="Hello">Y</p.components.Box>
```

Comment:
Isn't this invalid JSX since all custom components need to be upper case? Maybe change it to

```JSX
P => <P.components.Box ... />
```

Reply:
@mxstbr it's valid. custom components need to start with uppercase only if it's a variable name, for property access etc, it doesn't matter. example: https://codesandbox.io/s/ql0mzp2jl6